src/python/pants/core/goals/tailor_test.py

```diff
@@ -141,6 +141,28 @@ def test_rename_conflicting_targets(rule_runner: RuleRunner) -> None:
     )
 
 
+def test_root_targets_are_explicitly_named(rule_runner: RuleRunner) -> None:
+    rule_runner.create_file("foo.f90", "")
```

Comment:
No need to waste CI time fixing this, but FYI the content takes a default arg and you can leave that off.

Reply:
I prefer to be explicit anyway.